backend/src/main/java/edu/nus/market/controller/AccountController.java

```diff
@@ -23,31 +23,27 @@ public class AccountController {
     @Resource
     AccountService accountService;
 
-
-    @PatchMapping("/token")
-    public ResponseEntity<Object> resetPassword(@Valid @RequestBody ForgotPasswordReq forgotPasswordReq, BindingResult bindingResult){
-        //forget password and reset
+    @PostMapping("/token")
+    public ResponseEntity<Object> login(@Valid @RequestBody LoginReq loginReq, BindingResult bindingResult){
         if (bindingResult.hasErrors()) {
             return ResponseEntity.status(HttpStatus.BAD_REQUEST).body(new ErrorMsg(ErrorMsgEnum.INVALID_DATA_FORMAT.ErrorMsg));
         }
-        return accountService.forgotPasswordService(forgotPasswordReq);
+        return accountService.loginService(loginReq);
     }
 
-
-    @GetMapping("/me")
-
-    public Account getMyProfile(){
-        return accountService.getMyAccount(1);
+    @DeleteMapping("/token")
+    public ResponseEntity<Object> logout(@RequestHeader(value = "Cookie") String token){
+        return accountService.logoutService(token);
```

Comment:
This `token` does not need to passed down to service layer, because `logoutService` does not even use this parameter.

```suggestion
        return accountService.logoutService();
```